common/lib/xmodule/xmodule/x_module.py

```diff
@@ -402,7 +402,7 @@ def get_content_titles(self):
         else:
             return [self.display_name_with_default]
 
-    def get_children(self):
+    def get_children(self, location_filter=lambda location: True):
```

Comment:
Note to reviewers: using the `location_filter` in the `VideoSummaryOutline` reduces the number of calls to `get_block` by about half for the courses tested.